custom_components/tahoma/TahomaDataUpdateCoordinator.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
```

Comment:
Is the name of this file according to the python guidelines? Can we maybe just call it coordinator.py?